test/all.sh

```diff
@@ -5,7 +5,7 @@ rootDir="$(pwd -P)"
 
 # Deliberately test that the tests pass from a random symlink
 # to the source directory.
-tmpDir="$(mktemp -d /tmp/nvim-gdb-test.XXXXXX)"
+tmpDir="$(mktemp -d --tmpdir nvim-gdb-test.XXXXXX)"
```

Comment:
Unfortunately, doesn't work in MacOS: https://github.com/sakhnik/nvim-gdb/actions/runs/5026479605/jobs/9050865435?pr=189